src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -65,6 +66,11 @@ export default class NavigationAccordion extends React.Component {
           style={{ letterSpacing: '0.025em' }}
         >
           {label}
+          {external && (
+            <span className="ml3">
```

Comment:
Small styling suggestion: a slightly lighter color on the icon could help add distinction between the text and icon. And moving the icon 1px up will align it a little more with the text:

```suggestion
            <span className="ml3 color-darken50 relative" style={{top: '-1px'}}>
```

Current | Proposed
:---|:---
![localhost_8080_dr-ui_components_ (3)](https://user-images.githubusercontent.com/2180540/98882754-f02bf000-245a-11eb-9c42-0a542524ce4c.png) | ![localhost_8080_dr-ui_components_ (1)](https://user-images.githubusercontent.com/2180540/98882718-d4c0e500-245a-11eb-9673-38a395c7ed66.png)